src/data.cpp

```diff
@@ -170,9 +170,9 @@ NDF_HEADER(0x04, 17),
 /*06*/ OFFSET,          OPTIONS(2), 0x4A, 0x5A,
 /*07*/                  OPTIONS(1), 0x5D,
 /*08*/ OFFSET,          OPTIONS(1), 0x41,
-/*09*/ OFFSET,          OPTIONS(1), 0x06, 0x12,
+/*09*/ OFFSET,          OPTIONS(2), 0x06, 0x12,
```

Comment:
In the `NDF_HEADER` at line 167, you need to increment the `17` to `18` as that's essentially a version number. It's wrong with the previous function too, but I guess nobody has really noticed yet.

I do not know the exact details, but as far as I know this check data isn't actually read from this table but rather from a file on your disk. The version is just to overwrite it, when it has changed as far as I know.

Reply:
Thanks! Bumped version and amended commit.